tests/test_central_boiler.py

```diff
@@ -251,6 +251,7 @@ async def test_update_central_boiler_state_multiple(
     switch2 = MockSwitch(hass, "switch2", "theSwitch2")
     switch3 = MockSwitch(hass, "switch3", "theSwitch3")
     switch4 = MockSwitch(hass, "switch4", "theSwitch4")
+    switch5 = MockSwitch(hass, "switch5", "theSwitch5")
```

Comment:
Here I didn't test until 9 heaters because it was a lot of copy/paste/fix of code, maybe the up to 9 limit should be tested in a simpler test ?